### PR TITLE
word-search: Update tests to version 1.1.0(#1012)

### PR DESCRIPTION
**Pull request by wilson8507**
resolve issue #1012

### CODE REVIEW

File: exercises/word-search/word_search_test.py [hunk 1]
```diff
@@ -36,48 +137,179 @@ def test_vertical_words_top_to_bottom(self):
             self.example.search('ecmascript'),
             (Point(9, 0), Point(9, 9))
         )
+        self.assertEqual(
```
Comment (cmccandless) [inline exercises/word-search/word_search_test.py]: There are a lot of redundant and misplaced assertions. Please check that each assertion actually belongs under its test method (Example: checks for "clojure" in `self.example` should only appear in tests for horizontal words).

**Comment by wilson8507** [comment]
@cmccandless I've fixed it, please check if there is any problem. Thanks!

File: exercises/word-search/word_search_test.py [hunk 2]
```diff
@@ -2,9 +2,112 @@
 
 from word_search import WordSearch, Point
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.1.0
+
```
Comment (N-Parsons) [inline exercises/word-search/word_search_test.py]: Minor formatting point, but can you move the version comment down a line (ie. two blank lines before, one after).

**Comment by wilson8507** [comment]
@N-Parsons I've resolved it. Thank you for pointing out my carelessness!

**Comment by N-Parsons** [comment]
Perfect! Thanks, @wilson8507! I'm a bit picky about formatting and consistency :stuck_out_tongue:

**Pull request merged**